templates/scenarios/js/command-and-response/README.md

```diff
@@ -30,6 +30,7 @@ The app template is built using the TeamsFx SDK, which provides a simple set of
 >
 > - [Node.js](https://nodejs.org/), supported versions: 14, 16, 18
 > - An [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version after 5.0.0 or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
```

Comment:
- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher.
- [TeamsFx CLI](https://aka.ms/teamsfx-cli)

Reply:
I think TeamsFx CLI is not a required dependency if user has installed VSC Extension. Maybe following is more accuracy?

* [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [TeamsFx CLI](https://aka.ms/teamsfx-cli)